### ISSUE TITLE
error

### ISSUE THREAD

**Issue by MTalt-DrannaMT** [SheepTester/htmlifier#152]
I was converting my scratch project to HTML but found this error on the console:

Uncaught (in promise) DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The URL 'tw' is invalid.

**Comment by TheShovel** [comment]
Thats is because projects with the TurboWarp blocks can only be packaged in the Turbowarp packager. TW I am very sure is a extension name or id.

**Comment by SheepTester** [comment]
Yeah it seems like you're using an extension made for TurboWarp for the HTMLifier, which doesn't support TurboWarp stuff. You should use [TurboWarp Packager](https://packager.turbowarp.org/) instead
